apps/settings/js/modules/date_time.js

```diff
@@ -18,7 +18,6 @@ define(function(require) {
   var _kClockAutoAvailable = 'time.clock.automatic-update.available';
   var _kTimezoneAutoAvailable = 'time.timezone.automatic-update.available';
   var _kTimezone = 'time.timezone';
-  var _kUserSelected = 'time.timezone.user-selected';
```

Comment:
ugh! great catch. I convinced myself that it's not necessary anymore, but now I don't see a correlation. I'll revert this change.